samples/aws/AwsAppHost/UserService/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
```

Comment:
We shouldn't be using the docker files in this repo. We've using the build in container build support 😄